t/lib/t/MusicBrainz/Server/Controller/WS/2/DefaultFormat.pm

```diff
@@ -0,0 +1,91 @@
+package t::MusicBrainz::Server::Controller::WS::2::DefaultFormat;
+
+use JSON qw( decode_json encode_json );
+use Test::Deep qw( cmp_deeply );
+use Test::JSON import => [qw( is_valid_json is_json )];
+use Test::More;
+use Test::Routine;
+use Test::XML::SemanticCompare;
+
+with 't::Mechanize', 't::Context';
+
+test 'MBS-9258: deduction of fmt from Accept header' => sub {
```

Comment:
These tests look useful, but I don't think they test MBS-9258 at all, which was an issue with search requests, not lookups...right?

But I also don't think we have any tests related to search requests right now, and those would be more difficult to add, since you'd need to mock the parts that return data from the search server.

I'm okay with merging this if those aren't easy to add, but we should remove "MBS-9258" from this line if it doesn't test it.